dbms/src/Storages/DeltaMerge/DMVersionFilterBlockInputStream.h

```diff
@@ -220,7 +220,7 @@ class DMVersionFilterBlockInputStream : public IBlockInputStream
     // First calculate the gc_hint_version of every pk according to the following rules,
     //     see the comments in `calculateRowGcHintVersion` to see how to calculate it for every pk
     // Then the block's gc_hint_version is the minimum value of all pk's gc_hint_version
-    UInt64 gc_hint_version;
+    UInt64 gc_hint_version{};
```

Comment:
`std::numeric_limits<UInt64>::max()` may be a better init value for gc_hint_version @lidezhu ?